src/Test/L1/Mock/FakeAgentPluginManager.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Agent.Sdk;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+
+namespace Microsoft.VisualStudio.Services.Agent.Tests.L1.Worker
+{
+    public class FakeAgentPluginManager : AgentPluginManager
```

Comment:
Nit: I know `Fake` is sometimes used to name things like this. Personal preference is to be more descriptive and use something like `IntegrationTestAgentPluginManager`.

Reply:
I think that would take away from readability. There's a bunch of mocks in this folder, and prefixing everything with `IntegrationTest` will hide the important stuff in some viewers.

Plus the info is already in the filepath.